drivers: serial: Fix RF5x UART IRQ Mask values

ioannisg opened the pull request:
In uart_nrf5.c the IRQ masks for the TX READY and ERROR events are incorrect. This pull request fixes the mask values so they are according to the nRF51/nRF52832/nRF52840 product specification.

carlescufi commented:
@ioannisg Commit message is too long and does not follow the subsystem: topic formatting

cvinayak commented:
Commit message title is formated as:
<subsystem name>: <module, soc, board etc.>: Short title
<blank line>
<Detailed why, and how>
<blank line>
Signed-off-by: <use your nordic semi email id>

ioannisg committed:
drivers: serial: Fix nRF5x UART IRQ Mask values

In uart_nrf5.c the IRQ masks for the TX READY and ERROR events are
incorrect. This pull request fixes the mask values so they are
according to the nRF51/nRF52832/nRF52840 product specification.

Signed-off-by: Ioannis Glaropoulos <Ioannis.Glaropoulos@nordicsemi.no>
